setup.py

```diff
@@ -15,7 +15,7 @@
 # https://blog.ionelmc.ro/presentations/packaging/
 setup(
     name="sumy",
-    version="0.7.0",
+    version="0.8.0",
```

Comment:
Please don't do this. I have automatic script for bumping the version on all places. Please use interactive rebacse to remove it from here and also the next commits that is manipulating it.

Reply:
Sure! For curiosity, what do you use?

Reply:
[bumpversion](https://github.com/miso-belica/sumy/blob/dev/tasks.py#L30)